### PR TITLE
Update Fluentd to v1.10.2

### PR DESCRIPTION
**Pull request by QuentinBisson**
## Checklist

- [x] Update changelog in CHANGELOG.md.

@stone-z could we use dependabot to do that automatically? Are you aware of any other tool we could use?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v1.10.2]
```
Comment (MarcelMue) [inline CHANGELOG.md:11]: Just leave this line out for now and keep it as unreleased for testing?

Comment (MarcelMue) [reply CHANGELOG.md:11]: Would also be cool to keep the usual changelog thingy of `Added` / `Changed` / `Fixed`

Example https://github.com/giantswarm/aws-operator/blob/master/CHANGELOG.md

**Comment by stone-z** [comment]
Dependabot does support Dockerfiles :+1:

**Comment by QuentinBisson** [comment]
So how should I go about enabling it :D

**Comment by stone-z** [comment]
I went ahead and added it as dependabot had somehow been removed again. There is some more info in [fmt](https://github.com/giantswarm/fmt/blob/master/github/dependabot.md) if you need to tune configuration

**Comment by QuentinBisson** [comment]
Thanks a lot :)

File: CHANGELOG.md (hunk 1 above)
Comment (QuentinBisson) [reply CHANGELOG.md:11]: I updated the changelog and had fluentd start locally. I am not sure I can test it as the CI is only set-up on the master branch. Should I add a docker build/push latest on all the branches so I can test it on a cluster?

Comment (QuentinBisson) [reply CHANGELOG.md:11]: My bad, it is build on every commit :(

**Pull request closed without merging**